service/iot-service-samples/pnp-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/PnPNoComponentSample.java

```diff
@@ -0,0 +1,60 @@
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.service.devicetwin.*;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+public class PnPNoComponentSample {
```

Comment:
Is this the name we want to use?

Reply:
If it is targeting thermostat, wouldn't it be better to call it thermostat (Or the name that we used on Device)?

Reply:
Open to suggestions here. We were planning to add NoComponent and Component samples based on sync with Eric. This name seemed to point that out so I kept it this way.

Reply:
How do you differentiate Component and NoComponent? I do agree it might be nice to have the name Thermostat somewhere.

Reply:
Or maybe we can just have one sample and call it thermostat and a switch for Component/Nocomponent?

Reply:
Updated based on discussion to Thermostat